src/textarea/Textarea.unit.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import { mount, shallow } from 'enzyme'
+
+import EyeIcon from 'icon/eyeIcon'
+import Textarea from './Textarea'
+
+it('Should have the proper value.', () => {
+  const wrapper = shallow(<Textarea name="test" defaultValue="blabla" />)
+  const textarea = wrapper.find('textarea')
+  expect(wrapper.hasClass('kirk-textarea')).toBe(true)
+  expect(textarea.prop('value')).toBe('blabla')
+})
+
+it('Should have a valid button with title', () => {
+  const mockOnButtonClick = jest.fn()
+  const props = {
+    name: 'test',
+    defaultValue: 'blabla',
+    buttonTitle: 'button title',
+    buttonIcon: <EyeIcon />,
```

Comment:
nit: you could also add an "expect" for the icon , it's not tested elsewhere

Reply:
I added a check for the icon in the button. Thanks!